source/appModules/explorer.py

```diff
@@ -292,6 +292,15 @@ def event_NVDAObject_init(self, obj):
 		# Lets hide that
 		if windowClass=="msctls_progress32" and winUser.getClassName(winUser.getAncestor(obj.windowHandle,winUser.GA_PARENT))=="Address Band Root":
 			obj.presentationType=obj.presType_layout
+			return
+
+		if windowClass == "DirectUIHWND" and role == controlTypes.ROLE_LIST:
+			parent = obj.parent.parent.parent

```

Comment:
You need to check that the  intermediate parents are not None.  Perhaps put the whole thing in a:
```
if obj.parent and obj.parent.parent:
```
Note that as properties are cached for the current core cycle, this will not inappropriately fetch parents more than once.

Reply:
Done. Are there real life situations in which there are not at least two parents for an object, or is it only just in case?